etc/version.lock.json

```diff
@@ -76,8 +76,8 @@
   },
   "0a97b20f-4144-49ea-be32-b540ecc445de": {
     "rule_name": "Malware - Detected - Endpoint Security",
-    "sha256": "d4b0108faa80fc35468cc5cfbbaf48b4db4dad7d1373cf48388752568eb83c98",
-    "version": 5
+    "sha256": "adcd895329cc4d1c41bc4bf8b75404c838823731713fa11f3d3b671dd24cc31d",
+    "version": 4
```

Comment:
so this one went backwards to 4 because the rule was un-changed, because it was endgame.
this took me a second but makes sense

Reply:
also @paulewing should this rule be renamed to `- Elastic Endgame`?